### ISSUE TITLE
RightCurly: ALONE option doen't work for a singleline IF

### ISSUE THREAD

**Issue by Vladlis** [checkstyle/checkstyle#4044]
vlad@vlad-W65-67SB ~/Documents $ javac Input.java
vlad@vlad-W65-67SB ~/Documents $ java -jar checkstyle-7.6-all.jar -c my_check.xml Input.java
Starting audit...
Audit done.
vlad@vlad-W65-67SB ~/Documents $ cat Input.java 
```java
public class Input {
    void foo() {
        if (true) { getClass(); }
    }
}
```
vlad@vlad-W65-67SB ~/Documents $ cat my_check.xml 
```xml
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC
          "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
          "http://www.puppycrawl.com/dtds/configuration_1_3.dtd">

<module name = "Checker">
    <property name="charset" value="UTF-8"/>
    <property name="severity" value="warning"/>

    <module name="TreeWalker">
         <module name="RightCurly">
             <property name="option" value="alone"/>
         </module>
    </module>
</module>
```

Expected: violation on `if` as `option` is `alone`

**Comment by MEZk** [comment]
@romani 
I am on it

**Comment by romani** [comment]
fix is merged
